src/com/perl5/lang/perl/util/PerlRunUtil.java

```diff
@@ -74,6 +85,12 @@ public static GeneralCommandLine getPerlCommandLine(@NotNull Project project, @N
 			commandLine.addParameter("-I" + FileUtil.toSystemDependentName(includePath));
 		}
 
+		for (String path : getPerlLibraries(project, runProfile, scriptFile))
```

Comment:
I believe, there is no need to put this logic here. It relates to PerlRunProfileState only, so belongs there